pkg/lib/dialogs.jsx

```diff
@@ -106,7 +106,14 @@ export const WithDialogs = ({ children }) => {
     const [dialog, setDialog] = useState(null);
 
     const Dialogs = {
-        show: setDialog,
+        show: component => {
+            if (dialog !== null)
+                console.error("Dialogs.show() called for",
+                              JSON.stringify(component),
+                              "while a dialog is already open:",
+                              JSON.stringify(dialog));
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19595-20231109-125451-e2d79519-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Heh yes, that's the point :grin:

Reply:
Technically, `Dialogs.show(null)` is allowed and documented to be the same as `Dialogs.close()`. So we need to take that into account, or change the docs.

Reply:
Good point, thanks for spotting! Fixed.